CONTRIBUTING.md

```diff
@@ -165,8 +165,17 @@ any of the following conditions:
    submodule and re-exporting them from the original module. Must not include
    any code changes except to import paths. Requires absolutely no change to the
    public API.
+4. PR as previously had two ACKs, had minimal changes, and gets a single ACK
```

Comment:
has previously had?

Reply:
Oh, lol. `as` sorta works in a flowery way *puts on monocle* "I do say sir, a PR as [one which has] had previously two ACKs, shall get a single ACK".

But I think you're right that it was supposed to be `has`, which can be read in a normal tone.

I think it'd be fine to fix the typo and then one-ACK merge this, since the typo doesn't really affect the meaning, and for process changes it's really just concept ACKs that we're looking for.